src/validatePrTitle.test.js

```diff
@@ -107,6 +107,28 @@ describe('description validation', () => {
   it('does not validate the description by default', async () => {
     await validatePrTitle('fix: sK!"§4123');
   });
+  it('throws custom error', async () => {
```

Comment:
Maybe a bit more specific:

```suggestion

  it('uses the `subjectPatternError` if available when the `subjectPattern` doesn't match', async () => {
```